src/mlpack/methods/gmm/em_fit.hpp

```diff
@@ -118,6 +120,8 @@ class EMFit
   //! Modify the covariance constraint policy class.
   CovarianceConstraintPolicy& Constraint() { return constraint; }
 
+  //! Get the distribution.
```

Comment:
I guess this comment is indicative of the rest of the code that follows? Or is it something that can be removed?